UI/lib/dynatable.html

```diff
@@ -38,57 +38,60 @@
-				 type="hidden" 
+                                 type="hidden" 
              name=NAME
              value=ROW.${COL.col_id} } ?>
       <?lsmb- ELSE - ?>
       <td class="<?lsmb COL.col_id _ ' ' _ COL.class _ ' ' _ COL.type ?>">
           <?lsmb- IF TYPE == 'input_text' -?>
           <?lsmb 
                  PROCESS input element_data = {
-					  id=COL.col_id _ '-' _ ROWCOUNT
-					  type="text"
+                                          id=COL.col_id _ '-' _ ROWCOUNT
+                                          type="text"
                                           "data-dojo-type" = COL.data_dojo_type
-					  class=COL.class
-					  name=PFX _ COL.col_id _ '_' _ ROW.row_id
-					  value=ROW.${COL.col_id} } ?>
+                                          class=COL.class
+                                          name=PFX _ COL.col_id _ '_' _ ROW.row_id
+                                          value=ROW.${COL.col_id} } ?>
          <?lsmb- ELSIF TYPE == 'checkbox';
                    IF ROW.${COL.col_id};
                       CHECKED = "checked";
                    ELSE; 
                       CHECKED = '';
                    END;
                    ?> 
-	 <?lsmb PROCESS input element_data = {
-			  id=COL.col_id _ '-' _ ROWCOUNT
-			  type="checkbox"
-			  name=PFX _ COL.col_id _ '_' _ ROWCOUNT
-			  class=COL.class
-			  value=ROW.row_id
-			  checked=CHECKED } ?>
+         <?lsmb PROCESS input element_data = {
+                          id=COL.col_id _ '-' _ ROWCOUNT
+                          type="checkbox"
+                          name=PFX _ COL.col_id _ '_' _ ROWCOUNT
+                          class=COL.class
+                          value=ROW.row_id
+                          checked=CHECKED } ?>
          <?lsmb- ELSIF TYPE == 'radio' ?> 
          <?lsmb PROCESS input element_data = {
-					 id=COL.col_id _ '-' _ ROWCOUNT
-					 type="radio"
-					 name=PFX _ COL.col_id
-					 class=COL.class
-					 value=ROW.row_id
+                                         id=COL.col_id _ '-' _ ROWCOUNT
+                                         type="radio"
+                                         name=PFX _ COL.col_id
+                                         class=COL.class
+                                         value=ROW.row_id
                 checked=CHECKED } ?>
          <?lsmb- ELSIF TYPE == 'href'; 
                    HREF_SFX = COL.col_id _ "_href_suffix";
                    IF ROW.$HREF_SFX;
                       HREF = COL.href_base _ ROW.$HREF_SFX;
                    ELSE;
                       HREF = COL.href_base _ ROW.row_id;
-                   END
-          ?><a href="<?lsmb HREF ?>"><?lsmb ROW.${COL.col_id} ?></a>
+                   END;
+                   IF COL.href_target;
+                      HREF_TGT = ' target="' _ COL.href_target _ '"';
+                   END;
+          ?><a href="<?lsmb HREF ?>"<?lsmb HREF_TGT ?>><?lsmb ROW.${COL.col_id} ?></a>
```

Comment:
This block is the only functional change here; the rest are tabs-to-spaces conversion.